src/lib/components/svg-view/DraggableSVG.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+	import type { iPoint } from "$lib/interfaces/iPoint";
+	import { activeModel } from "$lib/globalState/activeModel";
+	import { scale } from "$lib/globalState/scaleStore";
+
+	export let position: iPoint;
+
+	let controller: AbortController;
+	let signal: AbortSignal;
+
+	//Sets up eventlisteners when the mouse is pressed down on the svg
+	function onPointerDown() {
+		controller = new AbortController();
+		signal = controller.signal;
+		window.addEventListener("pointermove", onPointerMove, { signal });
+		window.addEventListener("pointerup", onPointerUp, { signal });
```

Comment:
Pointervents cannot be treated exactly the same as Mouseevents. For example, I don't see an event listener for `pointercancel`. Did you read and understand the example I pointed you to? https://github.com/ECDAR-AAU-SW-P5/Ecdar-GUI-Web/blob/main/src/routes/%2Bpage.svelte

Reply:
@AlbertHald dette sortere under dig

Reply:
Personligt nej tak, men også ja tak @atjn

Reply:
is this solved now? @atjn